### PR TITLE
GHC-117: Operative Progress form

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/concepts/operative_progress.csv [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,Units
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,Pre-Op Diagnosis,,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,Misc,Text,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Lets use the FSN as 'Pre-Operative Diagnosis'

File: configuration/concepts/operative_progress.csv [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+f348e4b5-4124-4169-966a-2dff2a944566,,post-Op Diagnosis,,diagnóstico postoperatorio,diagnóstico postoperatorio,,Misc,Text,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Lets use the FSN as 'Post-Operative Diagnosis'

File: configuration/concepts/operative_progress.csv [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,Local,Local,Local,Local,,Misc,N/A,,,
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,ConvSet,N/A,,Gen; Mac; Local; Knee Block; Ax Block; Interscalene Block; IV Block; Loc//Sed,
+b5ab7bc0-bffb-4dc8-b832-2e1713f06944,,Estimated Blood Loss,EBL,Pérdida de sangre estimada,Pérdida de sangre estimada,,Misc,Numeric,,,mls
+64fe56e6-5b52-4cdb-adc2-b6d50bfdee63,,Complications,,Complicaciones,Complicaciones,,Misc,Text,,,
+08f1f638-b3c7-4040-8ae9-c95a5df59755,,Specimens,,especímenes,especímenes,,Misc,Text,,,
+59185273-c5b8-4e73-8ba2-c68710f6c507,,Drains,,Drenajes,Drenajes,,Misc,Text,,,
+a1da87f9-c4af-4598-accd-489268a2d325,,Condition upon Discharge from OR,,Condición al alta del quirófano,Condición al alta del quirófano,,Misc,Text,,,
+0934658e-fee3-44e3-a8f2-292dc3587f54,,Time of Procedure,,Tiempo de Procedimiento,Tiempo de Procedimiento,,Misc,Text,,,
+2554a66d-46fe-4f5f-813a-2b491716bddf,,Post Operative Instruction,,Instrucción postoperatoria,Instrucción postoperatoria,,Misc,Text,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Lets use the FSN as 'Post-Operative Instruction'

File: configuration/concepts/operative_progress.csv [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,Units
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,Pre-Op Diagnosis,,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,Misc,Text,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,post-Op Diagnosis,,diagnóstico postoperatorio,diagnóstico postoperatorio,,Misc,Text,,,
+27fe6714-0bc6-4435-adb0-818538abe42c,,Procedure,,Procedimiento,Procedimiento,,Misc,Text,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,Surgeon/ Assistant,,Cirujano/ Asistente,Cirujano/ Asistente,,Misc,Text,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Leave this as 'Assistants'

File: configuration/concepts/operative_progress.csv [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,General,Gen,General,General,,Misc,N/A,,,
+5392d074-eed3-4f88-a8e9-5bae333e7589,,Monitired Anesthesia Care,Mac,Atención anestésica supervisada,Atención anestésica supervisada,,Misc,N/A,,,
+84ba7929-3714-4023-843a-2b1814563207,,Knee Block,,bloque de rodilla,bloque de rodilla,,Misc,N/A,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,Axillary Block,Ax Block,Bloque Axilar,Bloque Axilar,,Misc,N/A,,,
+0f087406-b092-419c-a9ae-bd32c9e29a43,,Interscalene Block,,Bloque interescalénico,Bloque interescalénico,,Misc,N/A,,,
+20fafddb-bfd4-421e-acfb-639454741e94,,Intravenous Block,IC Block,Bloque intravenoso,Bloque intravenoso,,Misc,N/A,,,
+3dbb2d82-ee96-43a5-a905-0401c2d8b1a9,,Local anesthesia and Sedative,Loc//Sed,Anestesia local y Sedante,Ubicación//Sed,,Misc,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Let's change this to 'Local anesthesia and Sedation' and maintain it as the Short Name

File: configuration/concepts/operative_progress.csv [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,Local,Local,Local,Local,,Misc,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Let's change this to 'Local anesthesia'

File: configuration/concepts/operative_progress.csv [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,ConvSet,N/A,,Gen; Mac; Local; Knee Block; Ax Block; Interscalene Block; IV Block; Loc//Sed,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: The concepts you have added as members don't exist

File: configuration/concepts/operative_progress.csv [hunk 8]
```diff
@@ -0,0 +1,21 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,Units
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,Pre-Op Diagnosis,,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,Misc,Text,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,post-Op Diagnosis,,diagnóstico postoperatorio,diagnóstico postoperatorio,,Misc,Text,,,
+27fe6714-0bc6-4435-adb0-818538abe42c,,Procedure,,Procedimiento,Procedimiento,,Misc,Text,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,Surgeon/ Assistant,,Cirujano/ Asistente,Cirujano/ Asistente,,Misc,Text,,,
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,General,Gen,General,General,,Misc,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Maintain the SSN as the FSN

File: configuration/concepts/operative_progress.csv [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+5392d074-eed3-4f88-a8e9-5bae333e7589,,Monitired Anesthesia Care,Mac,Atención anestésica supervisada,Atención anestésica supervisada,,Misc,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: 'Monitored Anesthesia Care'

File: configuration/concepts/operative_progress.csv [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+84ba7929-3714-4023-843a-2b1814563207,,Knee Block,,bloque de rodilla,bloque de rodilla,,Misc,N/A,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,Axillary Block,Ax Block,Bloque Axilar,Bloque Axilar,,Misc,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Maintain the SSN as FSN

**Review by reagan-meant — CHANGES_REQUESTED** [review]
Add all the usual header metadata including _order.
Add 'Short Specified Name' for all concepts

**Comment by reagan-meant** [comment]
@basrem could you also address the points that were in the ticket description!

File: configuration/concepts/operative_progress.csv (hunk 10 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 9 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 8 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 7 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 6 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 5 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 4 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 3 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 2 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 1 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv [hunk 11]
```diff
@@ -0,0 +1,22 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Description:es,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,,Pre-Operative Diagnosis,Pre-Op Diagnosis,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,,Misc,Text,,,,,,,,,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,,post-Operative Diagnosis,post-Operative Diagnosis,Diagnóstico postoperatorio,Diagnóstico postoperatorio,,,Misc,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: We change to 'Post-Operative Diagnosis'

File: configuration/concepts/operative_progress.csv [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+27fe6714-0bc6-4435-adb0-818538abe42c,,,Procedure,Procedure,Procedimiento,Procedimiento,,,Misc,Text,,,,,,,,,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,,Assistants,Assistants,Asistentes,Asistentes,,,Misc,Text,,,,,,,,,,,
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,,General,General,General,General,,,Misc,N/A,,,,,,,,,,,
+5392d074-eed3-4f88-a8e9-5bae333e7589,,,Monitored Anesthesia Care,Monitored Anesthesia Care,Atención de anestesia supervisada,Atención de anestesia supervisada,,,Misc,N/A,,,,,,,,,,,
+84ba7929-3714-4023-843a-2b1814563207,,,Knee Block,Knee Block,bloque de rodilla,bloque de rodilla,,,Misc,N/A,,,,,,,,,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,,Axillary Block,Axillary Block,Bloque Axilar,Bloque Axilar,,,Misc,N/A,,,,,,,,,,,
+0f087406-b092-419c-a9ae-bd32c9e29a43,,,Interscalene Block,Interscalene Block,Bloque interescalénico,Bloque interescalénico,,,Misc,N/A,,,,,,,,,,,
+20fafddb-bfd4-421e-acfb-639454741e94,,,Intravenous Block,IV Block,Bloque intravenoso,bloque CI,,,Misc,N/A,,,,,,,,,,,
+3dbb2d82-ee96-43a5-a905-0401c2d8b1a9,,,Local anesthesia and Sedation,Local anesthesia and Sedation,Anestesia local y Sedación,Anestesia local y Sedación,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: We change to 'Local Anesthesia and Sedation'

File: configuration/concepts/operative_progress.csv [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,,Local anesthesia,Local anesthesia,Anestesia local,Anestesia local,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: We change to 'Local Anesthesia'

File: configuration/concepts/operative_progress.csv [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,,ConvSet,N/A,,,General; Monitored Anesthesia Care; Knee Block; Axillary Block; Interscalene Block; Intravenous Block; Local anesthesia and Sedation,,,,,,,,
+b5ab7bc0-bffb-4dc8-b832-2e1713f06944,,,Estimated Blood Loss,EBL,Pérdida de sangre estimada,LBE,,,Misc,Numeric,,,,,mls,,,,,,
+64fe56e6-5b52-4cdb-adc2-b6d50bfdee63,,,Complications,Complications,Complicaciones,Complicaciones,,,Misc,Text,,,,,,,,,,,
+08f1f638-b3c7-4040-8ae9-c95a5df59755,,,Specimens,Spcmns,especímenes,Spcmns,,,Misc,Text,,,,,,,,,,,
+59185273-c5b8-4e73-8ba2-c68710f6c507,,,Drains,Drains,Drenajes,Drenajes,,,Misc,Text,,,,,,,,,,,
+a1da87f9-c4af-4598-accd-489268a2d325,,,Condition upon Discharge from OR,Condition upon Discharge,Condición al alta del quirófano,Condición al alta,,,Misc,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Change to 'Condition upon discharge from OR'
And let's use the same for Short name

File: configuration/concepts/operative_progress.csv [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+0b21ca75-2fab-4c34-bf4c-f66ae2415829,,,Surgeon,Surgeon,Cirujano,Cirujano,,,Misc,Text,,,,,,,,,,,
+0934658e-fee3-44e3-a8f2-292dc3587f54,,,Time of Procedure,Time of Prdure,Tiempo de Procedimiento,tiempo de prdura,,,Misc,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Let's use 'Datetime' for the datatype

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/operative_progress.csv (hunk 15 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 14 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+b5ab7bc0-bffb-4dc8-b832-2e1713f06944,,,Estimated Blood Loss,EBL,Pérdida de sangre estimada,LBE,,,Misc,Numeric,,,,,mls,,,,,,
+64fe56e6-5b52-4cdb-adc2-b6d50bfdee63,,,Complications,Complications,Complicaciones,Complicaciones,,,Misc,Text,,,,,,,,,,,
+08f1f638-b3c7-4040-8ae9-c95a5df59755,,,Specimens,Spcmns,especímenes,Spcmns,,,Misc,Text,,,,,,,,,,,
+59185273-c5b8-4e73-8ba2-c68710f6c507,,,Drains,Drains,Drenajes,Drenajes,,,Misc,Text,,,,,,,,,,,
+a1da87f9-c4af-4598-accd-489268a2d325,,,Condition upon discharge from OR,Condition upon discharge from OR,Condición al alta del quirófano,Condición al alta,,,Misc,Text,,,,,,,,,,,
+0b21ca75-2fab-4c34-bf4c-f66ae2415829,,,Surgeon,Surgeon,Cirujano,Cirujano,,,Misc,Text,,,,,,,,,,,
+0934658e-fee3-44e3-a8f2-292dc3587f54,,,Time of Procedure,Time of Prdure,Tiempo de Procedimiento,tiempo de prdura,,,Misc,Datetime,,,,,,,,,,,
+2554a66d-46fe-4f5f-813a-2b491716bddf,,,Post-Operative Instruction,Post-Op Instruction,Instrucción Post-Operatoria,Instrucción postoperatoria,,,Misc,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: We change this to 'Post-Operative Instructions' for the FSN and short name

File: configuration/concepts/operative_progress.csv [hunk 17]
```diff
@@ -0,0 +1,22 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Description:es,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,,Pre-Operative Diagnosis,Pre-Op Diagnosis,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,,Misc,Text,,,,,,,,,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,,Post-Operative Diagnosis,Post-Operative Diagnosis,Diagnóstico postoperatorio,Diagnóstico postoperatorio,,,Misc,Text,,,,,,,,,,,
+27fe6714-0bc6-4435-adb0-818538abe42c,,,Procedure,Procedure,Procedimiento,Procedimiento,,,Misc,Text,,,,,,,,,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,,Assistants,Assistants,Asistentes,Asistentes,,,Misc,Text,,,,,,,,,,,
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,,General,General,General,General,,,Misc,N/A,,,,,,,,,,,
+5392d074-eed3-4f88-a8e9-5bae333e7589,,,Monitored Anesthesia Care,Monitored Anesthesia Care,Atención de anestesia supervisada,Atención de anestesia supervisada,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Change the short name to 'MAC'

File: configuration/concepts/operative_progress.csv [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+84ba7929-3714-4023-843a-2b1814563207,,,Knee Block,Knee Block,bloque de rodilla,bloque de rodilla,,,Misc,N/A,,,,,,,,,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,,Axillary Block,Axillary Block,Bloque Axilar,Bloque Axilar,,,Misc,N/A,,,,,,,,,,,
+0f087406-b092-419c-a9ae-bd32c9e29a43,,,Interscalene Block,Interscalene Block,Bloque interescalénico,Bloque interescalénico,,,Misc,N/A,,,,,,,,,,,
+20fafddb-bfd4-421e-acfb-639454741e94,,,Intravenous Block,IV Block,Bloque intravenoso,bloque CI,,,Misc,N/A,,,,,,,,,,,
+3dbb2d82-ee96-43a5-a905-0401c2d8b1a9,,,Local Anesthesia and Sedation,Local Anesthesia & Sedation,Anestesia local y Sedación,Anestesia local y Sedación,,,Misc,N/A,,,,,,,,,,,
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,,Local Anesthesia,Local Anesthesia,Anestesia local,Anestesia local,,,Misc,N/A,,,,,,,,,,,
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,,Misc,Coded,,General; Monitored Anesthesia Care; Knee Block; Axillary Block; Interscalene Block; Intravenous Block; Local anesthesia and Sedation,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/operative_progress.csv]: Add 'Local Anesthesia' concept as well to this set

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/operative_progress.csv (hunk 18 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 16 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

File: configuration/concepts/operative_progress.csv (hunk 17 above)
Comment (basrem) [reply configuration/concepts/operative_progress.csv]: Done

**Pull request merged**